muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/communicator/dao/VacationNotificationsDAO.java

```diff
@@ -0,0 +1,70 @@
+package fi.otavanopisto.muikku.plugins.communicator.dao;
+
+import java.util.Date;
+
+import javax.persistence.EntityManager;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaDelete;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Root;
+
+import fi.otavanopisto.muikku.model.users.UserEntity;
+import fi.otavanopisto.muikku.plugins.CorePluginsDAO;
+import fi.otavanopisto.muikku.plugins.communicator.model.VacationNotifications_;
+import fi.otavanopisto.muikku.plugins.communicator.model.VacationNotifications;
+
+public class VacationNotificationsDAO extends CorePluginsDAO<VacationNotifications> {
+
+  private static final long serialVersionUID = 3790128454976388680L;
```

Comment:
UID is in use. Easiest way to generate a new one is to remove this line and use eclipse helper on the class name (it should give option to generate new serialVersionUID)